process/tech-due-diligence-guidelines.md

```diff
@@ -0,0 +1,103 @@
+# Technical Due Diligence Guidelines
+
+This page provides guidelines to those leading or contributing to technical due diligence exercises performed by or on behalf of the Technical Oversight Committee of the CNCF.
+
+## Introduction
+
+Part of the evaluation process in deciding upon initial or continued
+inclusion of projects into the CNCF is a Technical Due Diligence
+('Tech DD') exercise.  Ultimately the voting members of the TOC will,
+on the basis of this and other information, vote for or against the
+inclusion of each project at the relevant time.
+
+## Leading a Technical Due Diligence
+
+### Primary Goals
+
+To enable the voting TOC members to cast an informed vote about a
+project, it is crucial that each member is able to form their own
+opinion as to whether and to what extent the project meets the agreed
+upon [criteria](https://www.cncf.io/projects/graduation-criteria/) for
+inception, incubation or graduation. As the leader of a DD, your job
+is to make sure that they have whatever information they need,
+succinctly and readily available, to form that opinion.
+
+As a secondary goal, it is in the interests of the broader CNCF
+ecosystem that there exists some reasonable degree of consensus across
+the community regarding the inclusion or otherwise of projects at the
+various maturity levels.  Making sure that the relevant information is
+available, and any disagreement or misunderstanding as to it's
+validity are ideally resolved, helps to foster this consensus.
+
+### Where to start
+
+* make sure you're clear on what the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/)
```

Comment:
Is it worth calling out the cloud native properties from the charter? https://www.cncf.io/about/charter/

Reply:
This shouldn't entirely replace the work of the sponsor, who should help the project craft their proposal, addressing issues like why/how it's a fit for cloud native, community and adoption metrics, etc.

Reply:
@quinton-hoole let's call that out?

Reply:
Yeah, agreed.  I'll add some clarification.